code/game/objects/items/nemesis_items.dm

```diff
@@ -0,0 +1,361 @@
+	if(!gloves || !istype(gloves))
+		return ..()
+
+	if(overloaded)
+		gloves.gain_charge(round(NEMESIS_CHARGE_PER_MINE / 2)) //Small refund
+	else
+		if(gloves.charge < NEMESIS_CHARGE_PER_MINE)
+			to_chat(user, "<span class='warning'>Your A.R.E.S. suit does not posess enough charge to overload [src]! ([gloves.charge]/[NEMESIS_CHARGE_PER_MINE])</span>")
+			return ..()
+
+		gloves.lose_charge(NEMESIS_CHARGE_PER_MINE)
+
+	overloaded = !overloaded
+	to_chat(user, "<span class='notice'>You [overloaded ? "overload [src]'s detection system" : "return [src]'s circutry back to the normal state"].</span>")
+	icon_state = "[overloaded ? "nemesis_trap" : "nemesis_trap_inactive"]"
+
+/obj/item/nemesis_mine/attack_hand(mob/living/user, list/modifiers)
+	if(!active)
+		return ..()
+
+	to_chat(user, "<span class='warning'>You attempt to disarm [src]...</span>")
+	if(!do_after(user, 7 SECONDS, src))
+		return
+	active = FALSE
+	icon_state = initial(icon_state)
+	update_icon()
+	anchored = FALSE
+	to_chat(user, "<span class='notice'>You successfully disarm [src].</span>")
+
+/obj/item/nemesis_mine/proc/trigger(mob/living/L)
+	var/turf/T = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle(T)
+	L.apply_damage(75, STAMINA)
+	to_chat(L, "<span class='userdanger'>You trigger [src] and it explodes in shower of sparks!</span>")
+	playsound(T, 'sound/magic/disable_tech.ogg', 50, TRUE)
+
+	if(glasses && ishuman(glasses.loc))
+		var/mob/living/carbon/human/H = glasses.loc
+		if(H.get_item_by_slot(ITEM_SLOT_EYES) != glasses)
+			return
+		glasses.say("Attention: Tactical energy mine in [get_area(T)] has detonated.")
+
+	qdel(src)
+
+/obj/item/nemesis_mine/proc/trigger_overload()
+	var/turf/T = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle/huge(T)
+	playsound(T, 'sound/magic/disable_tech.ogg', 50, TRUE)
+	for(var/mob/living/victim in range(1, T))
+		to_chat(victim, "<span class='userdanger'>You are hit by an energy blast!</span>")
+		victim.apply_damage(55, STAMINA)
+	qdel(src)
+
+/obj/item/nemesis_mine/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback, quickstart = TRUE)
+	. = ..()
+	icon_state = "trap_small"
+
+/obj/item/nemesis_mine/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	if(!..()) //If not caught, deploy
+		if(overloaded)
+			trigger_overload()
+			return
+		active = TRUE
+		icon_state = "trap_small"
+		update_icon()
+		anchored = TRUE
+		playsound(get_turf(src), 'sound/magic/blind.ogg', 50, TRUE)
+		if(ishuman(throwingdatum.thrower))
+			var/mob/living/carbon/human/H = throwingdatum.thrower
+			if(istype(H.get_item_by_slot(ITEM_SLOT_EYES), /obj/item/clothing/glasses/hud/security/sunglasses/nemesis))
+				glasses = H.get_item_by_slot(ITEM_SLOT_EYES)
+
+/obj/effect/temp_visual/nemesis_circle
+	name = "energy blast"
+	icon = 'icons/obj/nemesis.dmi'
+	icon_state = "trap_explosion"
+	duration = 8
```

Comment:
use SECONDS defines maybe

Reply:
It's the duration of the animation, using SECONDS wont make much sense

Reply:
you could say 0.8 SECONDS